common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -245,14 +246,18 @@ private static void loadLibrary(final ClassLoader loader, final String name, fin
             // Make sure the helper is belong to the target ClassLoader.
             final Class<?> newHelper = tryToLoadClass(loader, NativeLibraryUtil.class);
             loadLibraryByHelper(newHelper, name, absolute);
-        } catch (Exception e) { // Should by pass the UnsatisfiedLinkError here!
-            logger.debug("Unable to load the library: " + name + '.', e);
-            NativeLibraryUtil.loadLibrary(name, absolute);  // Fallback to local helper class.
+            return;
+        } catch (UnsatisfiedLinkError e) { // Should by pass the UnsatisfiedLinkError here!
+            logger.debug("Unable to load the library '" + name + "', trying other loading mechanism.", e);
```

Comment:
can we use the format with `{}` instead of manually building the string: `logger.debug("unable to load {}", name, e);`?